time_monkey/settings/base.py

```diff
@@ -125,3 +125,5 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
+
+AUTH_USER_MODEL = 'auth.User'
```

Comment:
This whole commit should be deleted since all of it's changes are discarded later.

Reply:
If you leave it without any AUTH_USER_MODEL it doesn't work itselfs. It looks strange, but I wanted to do it as simple as I can, step by step.